src/Common/Storage/Commands.Storage.Test/packages.config

```diff
@@ -2,7 +2,7 @@
 <packages>
   <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Common.Authentication" version="1.3.3-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.4.0-preview" targetFramework="net45" />
```

Comment:
Sounds good, in the future it will be better practice to send one PR with such cross-solution change, get it merged and then send another PR with the DataLakeAnalytics specific changes. This will make revision hstory more clean and also ease reviewing the change